.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=true
```

Comment:
I am intentionally keeping `.npmrc` and forcing `package-lock` on all loopback-next developers.

Personally, I have a global `.npmrc` file setting `package-lock` to `false`, because that's what I need when working on LB3 repositories. If there was no `.npmrc` file in loopback-next, then I would not be using package-lock functionality because of my global config.

IMO, it's best to not make any assumptions about the global npm configuration and explicitly tell npm that we want to use package-lock files.